babel.config.js

```diff
@@ -2,17 +2,12 @@ module.exports = {
   presets: [
     [
       '@babel/preset-env',
-      // @TODO: Do we need useBuiltIns?
-      // {
-      //   useBuiltIns: 'entry',
-      //   corejs: 2,
-      // },
+      {
+        useBuiltIns: 'entry',
+        corejs: 3,
```

Comment:
and this..... is replacing it?

Reply:
This plus the new imports at the top of `apps/drupal/index.js` and `apps/node-pl/index.js`